languages/swift/exercises/concept/tuples/.docs/hints.md

```diff
@@ -0,0 +1,17 @@
+## General
+
+The individual components of a tuple can be accessed by appending a _.n_ to the name of the tuple, where _n_ is the index of the component, statring with 0.
+
+## 1. Convert coordinates to polar
+
+- The individual components of a labeled tuple can also be accessed by appending a _.label_ to the name of the tuple, where _label_ is the label name of the component.
+- The `atan2(_:,_:)`function requires `Foundation` to be imported.
+- Floating point types in Swift have the [squareRoot() method][squareroot-docs].
+
+## 2. Merge two database records
+
+- Tuples can be [decomposed and assigned to individual constants or variables][tuples].
```

Comment:
From the student's perspective I find the link to the general tuples-documentation a little confusing because I need to grasp these examples and might have problems to find or understand it.
What about an inline example for decomposing a tuple from the introduction?

Reply:
This is generally not how our hints are structured. This mention alone should be enough for the student to re-check the introduction. I also found the general documentation quite well-written, but that's personal of course.